src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -17,6 +17,11 @@ const AppealsV2StatusPage = ({ appeal }) => {
   const { type, details } = status;
   const currentStatus = getStatusContents(type, details);
   const nextEvents = getNextEvents(type);
+  const form9Event = events.find(e => e.type === EVENT_TYPES.form9, null);
```

Comment:
Is it possible to get a form 9 more than once? I believe that's the case for some of the other events, because of the fact that multiple things can be on one appeal, and each thing can ping pong from one status to the other repeatedly.

Reply:
Oh, great point. I have no idea. @cmgiven?

Reply:
Multiple Form 9s are possible. It's generally the earliest. But check out department-of-veterans-affairs/vets.gov-team#8454.

Reply:
Awesome! I just made a comment in the code. I think we can leave this for now and fix it up properly in that ticket.